lib/Common/ColouredMsg.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2018, The TurtleCoin Developers
+// Copyright (c) 2018-2019, The Karbo Developers
+// Copyright (c) 2019, The Qwertycoin Developers
+// 
+// Please see the included LICENSE file for more information.
+
+#pragma once
+
+#include <Common/ConsoleTools.h>
+
+#include <iomanip>
+
+#include <ostream>
+
+#include <string>
+
+class ColouredMsg
+{
+    public:
+        ColouredMsg(std::string msg, Common::Console::Color colour) 
+                  : msg(msg), colour(colour) {}
+
+        ColouredMsg(std::string msg, int padding, 
+                    Common::Console::Color colour)
+                  : msg(msg), colour(colour), padding(padding), pad(true) {}
+
+
+        /* Set the text colour, write the message, then reset. We use a class
+           as it seems the only way to have a valid << operator. We need this
+           so we can nicely do something like:
+
+           std::cout << "Hello " << GreenMsg("user") << std::endl;
+
+           Without having to write:
+
+           std::cout << "Hello ";
+           GreenMsg("user");
+           std::cout << std::endl; */
+
+        friend std::ostream& operator<<(std::ostream& os, const ColouredMsg &m)
+        {
+            Common::Console::setTextColor(m.colour);
+
+            if (m.pad)
+            {
+                os << std::left << std::setw(m.padding) << m.msg;
+            }
+            else
+            {
+                os << m.msg;
+            }
```

Comment:
Minor code style suggestion:
```
if (m.pad) {
    os << std::left << std::setw(m.padding) << m.msg;
} else {
    os << m.msg;
}
```
This way `if-else` will look compact and easier to read.